reference/modules/form.md

```diff
@@ -36,6 +38,41 @@ instances for the case.
 
 Example: `pv:firstName`
 
+### Zaak
+
+<sup>Available since 11.2.0</sup>
+
+Prefix: `zaak`
+
+With the `zaak` prefix, all data inside
+a [zaak response](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/VNG-Realisatie/zaken-api/master/src/openapi.yaml#tag/zaken/operation/zaak_create)
+can be used to prefill the form. Some examples:
+
+| Source key         | Type of data                                            | Example                                                                         |
+|--------------------|:--------------------------------------------------------|---------------------------------------------------------------------------------|
+| zaak:url           | A URL reference to the zaak                             | https://example.com/zaken/api/v1/zaken/f4086828-b0b7-4e6c-a0ac-5ca1e44c5b06     |
+| zaak:uuid          | Zaak UUID                                               | f4086828-b0b7-4e6c-a0ac-5ca1e44c5b06                                            |
+| zaak:identificatie | The URL to the zaak                                     | ZK2023-00001                                                                    |
```

Comment:
"The URL to the zaak" 

This isn't correect is it?

Reply:
Changed it